src/thor/matrix_action.cc

```diff
@@ -50,6 +51,14 @@ std::string thor_worker_t::matrix(Api& request) {
     return matrix.SourceToTarget(options.sources(), options.targets(), *reader, mode_costing, mode,
                                  max_matrix_distance.find(costing)->second);
   };
+  if (costing == "bikeshare") {
```

Comment:
I don't think this is a proper way to implement the `bss_matrix`, I'd like to hear your opinions on how to make this properly...